test/runnable/test15371.d

```diff
@@ -0,0 +1,31 @@
+// EXTRA_SOURCES: imports/test15371.d
+
+/*
+TEST_OUTPUT:
+---
+
+---
+*/
+
+module test15371;
+
+import imports.test15371;
+import core.stdc.stdio;
+
+void main() {
+    auto test = new Test15371();
+    const auto hasMember = __traits(hasMember, test, "privateField");
```

Comment:
`auto` is not required.

Reply:
Habit from my C# days where any variable decl would be var if it had the type somewhere in the initializer.